Fireworks/Candidates/plugins/FWCandidateHGCalLegoProxyBuilder.cc

```diff
@@ -94,4 +94,4 @@ void FWCandidateHGCalLegoProxyBuilder::build(const reco::HGCalMultiCluster &iDat
 REGISTER_FWPROXYBUILDER(FWCandidateHGCalLegoProxyBuilder,
                         reco::HGCalMultiCluster,
                         "HGCal Multiclusters Lego",
-                        FWViewType::kLegoBit | FWViewType::kLegoHF);
+                        static_cast<int>(FWViewType::kLegoBit) | static_cast<int>(FWViewType::kLegoHFBit));
```

Comment:
Could the `static_cast` be removed now?

@alja Note that presently this PR includes the fix mentioned in https://github.com/cms-sw/cmssw/pull/45076#issuecomment-2136163268

Reply:
done